index.html

```diff
@@ -89,5 +93,147 @@ <h3 class="subtitle">I am a Software Developer</h3>
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+      <div class="project3 proj">
+        <h2 class="ptitle">Professional Art Printing Data</h2>
+        <p class="pbox">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+      <div class="project4 proj">
+        <h2 class="ptitle">Professional Art Printing Data</h2>
+        <p class="pbox">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+      <div class="project5 proj">
+        <h2 class="ptitle">Professional Art Printing Data</h2>
+        <p class="pbox">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+      <div class="project6 proj">
+        <h2 class="ptitle">Professional Art Printing Data</h2>
+        <p class="pbox">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="ptech">
+          <li class="tdisplay">html</li>
+          <li class="tdisplay">bootstrap</li>
+          <li class="tdisplay">Ruby</li>
+        </ul>
+        <button class="btstyle pjbutton" type="button">See Project</button>
+      </div>
+    </section>
+    <!--about me-->
+    <section class="aboutme">
+      <h1 class="title abtitle">About me</h1>
+      <p class="paragraph">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <button class="btstyle mainbt" type="button">Get My Resume</button>
```

Comment:
- [ ] Good job designing your buttons and applying the various button interactions 🔥 Kindly align this button as shown in the template so as to improve your user interface 👍 
- [ ] Also, add some space between your decoration image and the horizontal line so as to follow the template design 😄 
- Your design:
![Annotation 2022-03-19 062356](https://user-images.githubusercontent.com/85113567/159108728-ec97e211-c5e7-4323-b2fe-2385eca553f3.jpg)
- Figma design:
![Annotation 2022-03-19 062336](https://user-images.githubusercontent.com/85113567/159108734-cf5fc9f5-a62a-4a1a-98f3-5557c52b8b91.jpg)

Reply:
I aligned the button with the text and added space between the background image and the divisive line.
Thanks!